### ISSUE TITLE
AxaCheckbox lacks focused outline

### ISSUE THREAD

**Issue by salle18** [axa-ch/patterns-library#2080]
@axa-ch/checkbox 5.1.9

When using keyboard to navigate through form elements axa checkbox doesn't have any visual indication that it is being focused.

Further reading:
https://www.a11yproject.com/posts/2013-01-25-never-remove-css-outlines/

**Comment by markus-walther** [comment]
Yes, this is an oversight. The visible focus frame should be around the label or axa-checkbox, to include the label text and increase visual prominence when focussed.
